### PR TITLE
feat(gremlin): Add `complete_types` option

### LINKED ISSUE #10839: bug(Gremlin): Unsupported property value type: java.util.ArrayList
Reported [on Discord](https://discord.com/channels/872925471417962546/873606591335759872/1108646580900343818):

```
2023-05-18T06:40:27Z ERR failed to write batch error="Iterate: E0502: error in read loop, error message '{code:243 message:{"code":"UnsupportedOperationException","requestId":"31286efb-f251-4367-a589-e692f4ad41fc","detailedMessage":"Unsupported property value type: java.util.ArrayList"} attributes:map[]}'. statusCode: 243" duration=1903.2685 len=4 module=cli table=aws_s3_buckets
```

Destination Gremlin Plugin version - v2.0.2
Source AWS Plugin version - v17.2.0

### PR DESCRIPTION
**Pull request by disq**
Neptune doesn't support list types ([among other things](https://docs.aws.amazon.com/neptune/latest/userguide/access-graph-gremlin-differences.html)) so we need to represent lists as strings.

The option defaults to `false` (assuming we have more Neptune users than Apache TinkerPop users it was worded as such)

Fixes #10839

### CODE REVIEW

File: website/pages/docs/plugins/destinations/gremlin/overview.mdx [hunk 1]
```diff
@@ -78,3 +78,7 @@ This is the (nested) spec used by the Gremlin destination Plugin.
 - `max_concurrent_connections` (integer, optional. default: number of runtime processors)
 
   Maximum number of concurrent connections to the database.
+
+- `neptune_compatibility` (boolean, optional. default: true)
```
Comment (disq) [inline website/pages/docs/plugins/destinations/gremlin/overview.mdx:82]: Another option would be to have an option like `use_full_types` (which would default to `false`)

Comment (candiduslynx) [reply website/pages/docs/plugins/destinations/gremlin/overview.mdx:82]: `complete_types`?

Comment (disq) [reply website/pages/docs/plugins/destinations/gremlin/overview.mdx]: Good one, implemented in https://github.com/cloudquery/cloudquery/pull/10846/commits/b19f8c68ba639e9cbdf39c3056b187bb373166f8

**Review by candiduslynx — APPROVED** [review]
only 1 nit, otherwise good

**Review by candiduslynx — APPROVED** [review]

**Pull request merged**